lua/cfc_time/server/storage_options/mysql.lua

```diff
@@ -207,9 +297,37 @@ function storage:GetTotalTime( steamID, callback )
     query:start()
 end
 
-function storage:CreateSession( callback, steamID, sessionStart, sessionEnd, duration )
-    local newSession = self:Prepare( "newSession", callback, steamID, sessionStart, sessionEnd, duration )
-    newSession:start()
+function storage:CreateSession( callback, steamID, sessionStart, sessionEnd, sessionDuration )
+    local maxDuration = self.MAX_SESSION_DURATION
+    local sessionsCount = math.ceil( maxDuration / sessionDuration )
+    if sessionsCount == math.huge then sessionsCount = 1 end
```

Comment:
if you commit sams suggestion sessionCount will never be math.huge
maybe do something like `sessionCount = math.max(1, sessionCount)` instead because `n/math.huge == 0`